tasks/Functions-client/buildRequestJSON.js

```diff
@@ -0,0 +1,99 @@
+const { simulateRequest, buildRequest, getRequestConfig } = require("../../FunctionsSandboxLibrary")
+const { networkConfig } = require("../../network-config")
+const readline = require("readline-promise").default
+const axios = require("axios")
+const fs = require("fs")
+
+task("functions-build-request", "Creates a JSON file with Functions request parameters")
```

Comment:
Is this necessary to have? It might be confusing to have both this and our tooling having a send-request command. Also it could go in a separate PR as it doesn't seem to be required for offchain secrets, right?

Reply:
I do think it is useful.  I personally use it a lot and if users want to use some alternate tool for deploying and interacting with contracts, it is also useful.

However, I would support keeping this feature (in a separate PR), just because it is something I personally use frequently.